frontend/src/components/CardTitle/CardTitle.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Typography,} from "@material-ui/core";
+import {APP_NAME} from "../../constants/constants";
+import {VerticalSpace} from "../Shared/VerticalSpace"
+import {Centered} from "../Shared/Centered";
+import styled from "styled-components";
+
+export type TitleProps = {
+    readonly title: string | undefined;
+};
+
+const CardTitleComponent = (props: TitleProps | undefined) => {
+    const titleComponent = (
+        <>
+            <Centered>
+                <Typography component="h5" variant="h5">
+                    {props?.title || APP_NAME}
+                </Typography>
+                <VerticalSpace height="2rem"/>
+            </Centered>
+        </>
+    );
+
+    return (
+        titleComponent
+    );
+}
+
+export default styled(({ ...otherProps }) => (
+    <CardTitleComponent {...otherProps}   />
+))`
+     marginBottom: "20px",
```

Comment:
Nie masz tutaj podpowiadania CSSa?

![image](https://user-images.githubusercontent.com/18637116/112543603-479d7d80-8db6-11eb-9ffb-1fef792c9593.png)

Powinienes tutaj pisac normalnie jak w CSS:

![image](https://user-images.githubusercontent.com/18637116/112543629-51bf7c00-8db6-11eb-9f95-d0418c567876.png)


Nie wiem czy to dobre dawac margin na stale i to jeszcze w pixelach. Chyba bym to zostawił komponentowi, który będzie gdzieś ten Title wstawiał.

Reply:
mi też nie podpowiada

Reply:
![image](https://user-images.githubusercontent.com/18637116/112616680-502c9d00-8e24-11eb-879d-2b0c3e1ed66a.png)
Zainstalujcie sobie :)